xrpl/core/binarycodec/types/serialized_type.py

```diff
@@ -14,38 +14,21 @@ def __init__(self: SerializedType, buffer: bytes = bytes()) -> None:
 
     @classmethod
     @abstractmethod
-    def from_parser(
+    def from_parser(  # noqa: D102
         cls: Type[SerializedType],
         # TODO: Resolve Any (can't be ``BinaryParser`` because of circular imports)
         parser: Any,
         # length_hint is Any so that subclasses can choose whether or not to require it.
         length_hint: Any,
     ) -> SerializedType:
-        """
-        Constructs a new SerializedType from a BinaryParser.
-
-        Args:
-            parser: The parser to construct a SerializedType from.
-            length_hint: The number of bytes to consume from the parser.
-
-        Raises:
-            NotImplementedError: Always.
-        """
-        raise NotImplementedError("SerializedType.from_parser not implemented.")
+        pass
```

Comment:
Why the switch from `NotImplementedError` to `pass`?

Reply:
it's not necessary. since it's an `abstractmethod` it will raise if you try to instantiate anything that doesn't have it implemented. so i wanted to make all our `abstractmethod` stubs the same. some were using `pass` and others were raising